paper/paper.md

```diff
@@ -21,158 +21,63 @@ authors:
     orcid: 0000-0003-4589-9809
     affiliation: 1
 affiliations:
-  - name: DACS Chair, HPI, Digital Engineering Faculty, University of Potsdam, Potsdam, Germany
+  - name: Hasso-Plattner-Institute for Digital Engineering, Digital Engineering Faculty, University of Potsdam, Potsdam, Germany
     index: 1
 date: 30 June 2022
 bibliography: paper.bib
 ---
 
 # Summary
 
-It is crucial to have a good understanding of the datasets that are being worked with. For static datasets, users can explore them once, and document their findings for other team members to consume.
-For frequently changing datasets, this is not enough.
-After exploring, analysing and documenting it once, an understanding needs to be continously verified to be up-to-date, and adjusted it when it's not.
+It is crucial to have a good understanding of the datasets that are being worked with. For static datasets, users can explore the dataset once, and document their findings for other team members to consume.
+For frequently changing datasets, this may not be sufficient.
+After exploring, analysing and documenting it once, an understanding needs to be continously verified to be up-to-date, and adjusted when it is not.
 To address these problems, we developed `datapact`, a Python library that provides an ergonomic API for running assertions on DataFrames.
 
+`datapact` is available under MIT license at https://github.com/skn0tt/datapact.
+
 # Statement of need
 
 When working with continually changing datasets, e.g. occupancy statistics for intensive care units during a pandemic, dataset shift (@moreno2012unifying) will occur frequently.
 Sometimes because of human errors in data entry, sometimes because of bugs, errors or outages in the data delivery, and sometimes because there are changes in the underlying data.
```

Comment:
I would suggest rewriting this sentence to avoid so many "sometimes". For example:
```suggestion
Some common causes of data shift are human errors in data entry, bugs, errors or outages in the data delivery, or simply changes in the underlying data.
```

Reply:
love it!